sceptre/cli.py

```diff
@@ -295,9 +319,20 @@ def launch_stack(ctx, environment, stack):
 
     Creates or updates ENVIRONMENT/STACK.
     """
-    env = get_env(ctx.obj["sceptre_dir"], environment, ctx.obj["options"])
-    response = env.stacks[stack].launch()
-    if response != StackStatus.COMPLETE:
+    try:
+        env = get_env(ctx.obj["sceptre_dir"], environment, ctx.obj["options"])
+        if not env.stacks.get(stack, None):
```

Comment:
Not using get will throw an exception if there is no key, this would require an extra exception line with error message handling. Where can I refer to project conventions you mention?

Reply:
`_get_output_value()` in `stack_output.py`

Reply:
Resolved